guides/common/modules/proc_updating-an-scc-account.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Updating_an_SCC_Account_{context}"]
+= Updating an SCC Account
+
+Use the following procedure to update your SCC account in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Updating_an_SCC_Account_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *SUSE Subscriptions*.
+. Select your SCC account.
+. Update your SCC account as required.
```

Comment:
```suggestion
. Update your SCC account as required.
```
Can this be `Select the SCC accounts...` instead of Update? We are also using the same context - `update` in the next point.

Reply:
I don't understand what you mean. Can you please elaborate?

Reply:
`Select the SCC account as required`. At this level, end-users are not taking any action to update, right?

Reply:
I am not sure what you mean. Did my last change fix it?

Reply:
Yes, it is better :+1: 
LGTM :+1: